### ISSUE TITLE
KITTI Evaluation

### ISSUE THREAD

**Issue by SamChuah** [JiaRenChang/PSMNet#146]
Hi, 

Just wondering if my understanding of how you evaluate your KITTI results is correct. 

You split the KITTI training sets (200) into two subsets: training (160) and testing (40). Then you use the model trained on SceneFlow + KITTI training subset(160) to perform inference on the testing set (40). You then use the results and KITTI's matlab code to calculate the error. 

Is my understanding correct? 

Thank you

**Comment by zhiwenfan** [comment]
I think the author put the test result(40 images) on KITTI2015 using the model trained on the training set(160 images).
But the final model put on Google drive is trained using the all training set (200 images).

**Comment by SpadeLiu** [comment]
> I think the author put the test result(40 images) on KITTI2015 using the model trained on the training set(160 images).
> But the final model put on Google drive is trained using the all training set (200 images).

I agree with you. I test the model offered by the author on validation set (40 images) and found the 3-px error is much smaller. And I don't know the reason, maybe you are right.
